parser/src/schema/body.rs

```diff
@@ -14,6 +14,12 @@ pub(crate) enum Body {
     Json(String),
     #[serde(alias = "xml")]
     Xml(String),
+    #[serde(alias = "pdf")]
+    Pdf(String),
+    #[serde(alias = "png")]
+    Png(String),
+    #[serde(alias = "jpeg")]
```

Comment:
```suggestion
    #[serde(alias = "jpg", "jpeg")]
```

Let's also support jpg as a way to invoke